### PR TITLE
Update documentation in tutorial and merlin server

### PR DESCRIPTION
**Pull request by ryannova**
Changed tutorial documentation to encourage users to merlin server rather than installing their own redis server and configuring it manually. 

Added command summary for merlin server in merlin_commands.rst

### CODE REVIEW

**Comment by lucpeterson** [comment]
Docs preview: https://merlin--378.org.readthedocs.build/en/378/

File: docs/source/merlin_specification.rst [hunk 1]
```diff
@@ -227,9 +227,9 @@ see :doc:`./merlin_variables`.
        label   : ITER.%%
 
   ####################################
-  # Merlin Block (Required)
+  # Merlin Block (Optional)
```
Comment (koning) [inline docs/source/merlin_specification.rst:230]: The merlin block is still required, just the user part is optional. Sorry about the confusion.

Comment (ryannova) [reply docs/source/merlin_specification.rst]: Sorry when I made the changes for the documentation I branched off a WIP pull request #376. I have rebased this PR to only include changes related to the updated documentation. Please see #376 for the most recent changes.

**Comment by lucpeterson** [comment]
I added a PR to stick onto this branch that includes a few changes to get the docs to build without warning

**Comment by ryannova** [comment]
@lucpeterson I pulled your docs changes to my branch, which helps to build without error. Thanks

File: docs/source/merlin_commands.rst [hunk 2]
```diff
@@ -400,6 +400,35 @@ The ``--workers`` option allows you to pass in a regular expression of names of
 
 The only currently available option for ``--task_server`` is celery, which is the default when this flag is excluded.
 
+Hosting Local Server (``merlin server``)
```
Comment (lucpeterson) [inline docs/source/merlin_commands.rst:403]: underline this so it's a heading:

Hosting a Local Server (``merlin server``)
------------------------------------------

**Review by lucpeterson — APPROVED** [review]

**Comment by lucpeterson** [comment]
fix the conflicts and LGTM

**Pull request merged**